Office 365/o365/ingest/parser.yml

```
@@ -565,6 +566,10 @@ stages:
           office365.alert.status: "{{json_event.message.Status}}"
           rule.id: "{{json_event.message.PolicyId}}"
 
+      - set:
+          data.user.email: "{{parse_data.ParsedData.f3u}}"
```

`user.email` seems to be a more suitable field (cf [ECS specification](https://www.elastic.co/guide/en/ecs/8.11/ecs-user.html#field-user-email)):
```suggestion
          user.email: "{{parse_data.ParsedData.f3u}}"
```

---

I wanted to differentiate user.email from this one because in this case, the user is "SecurityComplianceAlerts". Maybe should I set it to "office365.user.email" as you said?